tests/runtime/builtin

```diff
@@ -30,6 +30,10 @@ NAME cpu
 PROG i:ms:1 { printf("SUCCESS %lu\n", cpu); exit(); }
 EXPECT_REGEX SUCCESS [0-9]+
 
+NAME ncpus
+PROG BEGIN { printf("ncpus = %ld\n", ncpus); exit(); }
+EXPECT ncpus = {{EXPECT_NCPUS}}
```

Comment:
If the assertion of exact number of cpus is too clunky, we can just do a regex test that the output is greater than 0 e.g. `^[1-9][0-9]*$`